pkg/adaptation/suite_test.go

```diff
@@ -470,13 +470,13 @@ func (m *mockPlugin) onClose() {
 	}
 }
 
-func (m *mockPlugin) Configure(cfg, runtime, version string) (stub.EventMask, error) {
+func (m *mockPlugin) Configure(_ context.Context, cfg, runtime, version string) (stub.EventMask, error) {
```

Comment:
why the undeclared param.. for ctx?

Reply:
mockPlugin never uses or passes the context on to any test-case specific handlers.